Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -234,7 +234,7 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
             dateCorrection: dateCorrection,
             url: command.url,
             httpMethod: command.httpMethod,
-            isFirstPartyResource: command.isFirstPartyRequest,
+            isFirstPartyResource: dependencies.firstPartyURLsFilter.isFirstParty(string: command.url),
```

Comment:
Do we still need `isFirstPartyResource` here? 

The `RUMResourceScope` also keeps `dependencies` property, so we could use `dependencies.firstPartyURLsFilter` [here](https://github.com/DataDog/dd-sdk-ios/blob/develop/Sources/Datadog/RUM/RUMMonitor/Scopes/RUMResourceScope.swift#L279) and [here](https://github.com/DataDog/dd-sdk-ios/blob/develop/Sources/Datadog/RUM/RUMMonitor/Scopes/RUMResourceScope.swift#L291)

Reply:
Or directly in [here](https://github.com/DataDog/dd-sdk-ios/blob/develop/Sources/Datadog/RUM/RUMMonitor/Scopes/RUMResourceScope.swift#L72)